installer/x86_64/install.sh

```diff
@@ -106,6 +106,11 @@ fi
 if [ "$install_env" != "build" ]; then
     onie_dev=$(blkid | grep ONIE-BOOT | head -n 1 | awk '{print $1}' |  sed -e 's/:.*$//')
     blk_dev=$(echo $onie_dev | sed -e 's/[1-9][0-9]*$//' | sed -e 's/\([0-9]\)\(p\)/\1/')
+
+    # check if we have an nvme device
+    blk_suffix=
+    echo $blk_dev | grep -q nvme0 && blk_suffix="p"
```

Comment:
>nvme0 [](start = 28, length = 5)

Suggest grep regex `^nvme\d`

Reply:
$blk_dev is starting with "/dev" so the ^ is not correct in this case.
as for the "\d" it is not working in POSIX regular expressions, it is possible to use [[:digit:]]
I think the current one is better don't you think?
The same approach used in ONIE, you can check this PR:
https://github.com/opencomputeproject/onie/commit/ed5108d023d23a876767f0311a8bc43d67d34bd6